wherehows-web/app/controllers/datasets/dataset.ts

```diff
@@ -219,4 +233,16 @@ export default class DatasetController extends Controller {
     const fromUpstream = get(this, 'isPolicyFromUpstream');
     set(this, 'compliancePolicyHasDrift', !fromUpstream && hasDrift);
   }
+
+  /**
+   * Triggered when the ownership information changes, will alert the user on the tab with a red dot if
+   * the current state of the dataset doesn't match the rules set out for the dataset ownership
+   * @param ownersNotConfirmed - Whether or not the owners for the dataset meet the requirements
+   */
+  @action
+  setOwnershipRuleChange(ownersNotConfirmed: boolean) {
+    once('afterRender', () => {
```

Comment:
Minor comment, does this need to be in the `afterRender` queue, or since it's an action that will be invoked from a child component, then just queueing for the next run loop should suffice?
```js
run(() => next(this, () => set(this, ...))));
```